src/components/CRow.vue

```diff
@@ -1,17 +1,29 @@
 <template>
   <tr class="border-b border-gray-200 hover:bg-blue-100">
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal">{{ priority }}</pre>
+    </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ message_id }}</pre>
     </td>
     <td class="border px-4 py-2 font-semibold text-sm " :style="{ color: getColorState() }">
-      <pre class="text-xs whitespace-normal">{{ name }}</pre>
+      <pre class="text-xs whitespace-normal">{{ nameState }}</pre>
     </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ args }}</pre>
     </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ kwargs }}</pre>
     </td>
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal">{{ enqueued_datetime }}</pre>
```

Comment:
i would prefer the following colums.

    if it's pending, the duration since it's enqueuing (enqueued x min ago)
    if t's started, the duration since it's started (started x min ago)
    if it's Error or success, the execution time,
    if it's canceled display nothing.
For all operations regarding dates use date fns. https://date-fns.org/

for example to display x minutes ago https://date-fns.org/v2.12.0/docs/formatDistanceToNowStrict

Reply:
Leave this columns, but add  wait_time and and execution_time.